extensions/wikia/Search/classes/Services/ESFandomSearchService.php

```diff
@@ -17,6 +17,7 @@ class ESFandomSearchService extends AbstractSearchService {
 	const STORIES_IMAGE_URL_KEY = 'image';
 	const STORIES_URL_KEY = 'url';
 	const STORIES_COUNT_MAX = 6;
+	const STORIES_SEARCH_TEMPLATE = 'fandom_stories';
```

Comment:
I wonder if there is going to be a lot of work around this module. I would say let's add the WF variable if there is future work involved.